demo_drivers/bifurcation_tracking/adaptive_hopf_with_separate_meshes.cc

```diff
@@ -60,12 +60,11 @@ namespace Global_Parameters
  // Rotation ratio
  double Alpha=0.0;
 
- // Control Flag that will read in the eigenfunction from disk
- // to avoid a length eigenvalue calculation while testing
- bool Read_in_eigenfunction_from_disk = true;
-
  //Set the normalisation for the eigenfunction
  std::complex<double> Eigenfunction_normalisation(1.0,0.0);
+ 
+ // Control Flag that will read in the eigenfunction from disk
+ bool Read_in_eigenfunction_from_disk = true;
```

Comment:
Comment is a bit loose. The control flag doesn't read in the eigenfuction. Suggest.

"// Control flag used to determine whether the eigenfunction is read from disk. 
 // Reading from disk avoids lengthy eigenvalue calculations in the self-tests"

Reply:
changed, commited and pushed.